Library/Homebrew/test/support/fixtures/cask/Casks/livecheck/livecheck-extract-plist.rb

```diff
@@ -0,0 +1,16 @@
+cask "livecheck-extract-plist" do
+  version "1.2.3"
+  sha256 "78c670559a609f5d89a5d75eee49e2a2dab48aa3ea36906d14d5f7104e483bb9"
+
+  url "file://#{TEST_FIXTURE_DIR}/cask/caffeine-suite.zip"
+  name "With Installer Manual"
+  desc "Cask with a manual installer"
+  homepage "https://brew.sh/"
+
+  livecheck do
+    url "file://#{TEST_FIXTURE_DIR}/cask/caffeine-incl-plist.zip"
```

Comment:
```suggestion
    url "file://#{TEST_FIXTURE_DIR}/cask/caffeine-with-plist.zip"
```

since some other cask fixtures already use `with` in their names.